### DISCUSSION TITLE
What to do when upload fails/the program runs abnormally/the device port is not found? | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#978] [Q&A]
# What to do when upload fails/the program runs abnormally/the device port is not found? | Seeed Studio Wiki

XIAO-FAQ

https://wiki.seeedstudio.com/noport_upload_fails/

**Comment by GitHub-Karl** [comment]
BootLoader mode doesn't work by pressing boot button while connecting. Is there another method?

**Reply by MatthewJeffson** [reply]
May I ask what XIAO are you using and if possible proving the images for us?

**Comment by GitHub-Karl** [comment]
This method does not work, therefore I ask for another method

 

Von: giscus[bot] ***@***.*** 
Gesendet: Sonntag, 25. Februar 2024 17:11
An: Seeed-Studio/wiki-documents ***@***.***>
Cc: GitHub-Karl ***@***.***>; Comment ***@***.***>
Betreff: [Seeed-Studio/wiki-documents] What to do when upload fails/the program runs abnormally/the device port is not found? | Seeed Studio Wiki (Discussion #978)

 


What to do when upload fails/the program runs abnormally/the device port is not found? | Seeed Studio Wiki


XIAO-FAQ

https://wiki.seeedstudio.com/noport_upload_fails/

—
Reply to this email directly, view it on GitHub <https://github.com/Seeed-Studio/wiki-documents/discussions/978> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AT5OHTTKJYKJGLNYNZMEDFLYVNPBPAVCNFSM6AAAAABDY7GPSWVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGI3DSNJSGE> .
You are receiving this because you commented.Message ID: ***@***.***>



-- 
Diese E-Mail wurde von Avast-Antivirussoftware auf Viren geprüft.
www.avast.com

**Comment by GitHub-Karl** [comment]
![SeeedStudio-XIAO-ESP32S3-Sense](https://github.com/Seeed-Studio/wiki-documents/assets/83551182/2380f02d-7e0c-4ee1-a7eb-b76277d75b67)
I've made a foto (above I hope
![SeeedStudio-XIAO-ESP32S3-Sense](https://github.com/Seeed-Studio/wiki-documents/assets/83551182/8647934b-9c7d-4bbc-bcc4-f484658299a0)
I've made a foto (above I hope it is visible). I had used it with the camera (not to be seen) and it had worked before.

**Comment by MatthewJeffson** [comment]
This board is XIAO ESP32S3 and enter the BootLoader mode should be like this: ![](https://files.seeedstudio.com/wiki/SeeedStudio-XIAO-ESP32S3/img/15.gif)
When you found it not working, the board might be broken and if the board is under the warranty period you can ask our tech support for help: fae@seeed.io or techsupport@seeed.cc

**Comment by masteclab** [comment]
Error, the device port is not found
![imagen](https://github.com/Seeed-Studio/wiki-documents/assets/26703220/b6952bf1-e739-43b2-85e5-027851521cd9)

This is the configuration:
![imagen](https://github.com/Seeed-Studio/wiki-documents/assets/26703220/e148e39f-0ef8-4d02-90e8-a9e1a460d6a4)
